pkg/node-servant/convert/convert.go

```diff
@@ -64,15 +60,6 @@ func (n *nodeConverter) validateOptions() error {
 	return nil
 }
 
-func (n *nodeConverter) preflightCheck() error {
```

Comment:
why we delete those?
and how can we make sure users do preflight before do convert?

or, what if we put core preflight capability in `convert`?

Reply:
There are two reasons for deletion:
(1) The node servant mainly serves for yurtctl convert, it is rare for users to use node-servant to convert nodes individually.
(2) In order to increase the conversion success rate of yrutcl convert, yurtctl convert will perform preflight check and  issue node servant preflight check job. If the check is successful, the cloud will perform the conversion and issue node servant convert job. If add a check part to the node servant, the check part will be executed twice. It seems a bit cumbersome.
So I remove the check part in node-servant convert.
How about marking it in the usage document: Perform node-servant preflight check before executing node-servant convert?

Reply:
> (1) The node servant mainly serves for yurtctl convert, it is rare for users to use node-servant to convert nodes individually.

I think it's not about we use it individually or not. 
I only think node-servant should include preflight check capability,
and export its capability to the upper ( `yurtctl convert` level )

> (2) In order to increase the conversion success rate of yrutcl convert, 

is this be noticed by users? 
I think end-user should not notice we have a preflight command if everything is right.
The end-user doesn't need to execute this manually I think.

what do you say?

Reply:
> I think it's not about we use it individually or not...

Yes, considering the usability of users, it is better that convert include preflight check .So yurtctl convert does just that.
But in node-servant, we split it into two parts, one for preflight check and one for conversion to cooperate with the use of yurtctl convert.
If the node-servant convert contains check and also exposes the check capability, then yurtctl convert will call twice repeated checks (as mentioned above) during conversion.

> is this be noticed by users?

If the user uses yurtctl convert, the preflight check content is included in the convert, so the user uses the same way as before.
If the user uses node-servant convert, preflight check and convert are two commands, so the user needs to execute preflight check first, and then execute convert, which needs to be stated in the document.